upup/pkg/kutil/delete_cluster.go

```diff
@@ -565,6 +565,10 @@ func DeleteSecurityGroup(cloud fi.Cloud, t *ResourceTracker) error {
 		}
 		response, err := c.EC2().DescribeSecurityGroups(request)
 		if err != nil {
+			if awsup.AWSErrorCode(err) == "InvalidGroup.NotFound" {
```

Comment:
Can this string: `"InvalidGroup.NotFound"` be replaced with a value in AWS api?  My concern is that they refactor the message, i.e. change their contract.  Our code would not be smart enough to pick up the changes.